books/RayTracingInOneWeekend.html

```diff
@@ -2844,18 +2844,18 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ Highlight
     class lambertian : public material {
       public:
-        lambertian(const color& a) : albedo(a) {}
+        lambertian(const color& c) : c(c) {}
 
         bool scatter(const ray& r_in, const hit_record& rec, color& attenuation, ray& scattered)
         const override {
             auto scatter_direction = rec.normal + random_unit_vector();
             scattered = ray(rec.p, scatter_direction);
-            attenuation = albedo;
+            attenuation = c;
             return true;
         }
 
       private:
-        color albedo;
+        color c;
```

Comment:
My preference would be `col` (or `colour` if we want to make people made). 
One less single character variable to remember.

x, y, z, Q, u, v, etc are all used in formulas.

Reply:
After discussion in Slack, reverted back to "albedo". See also #1430.